spec/honeycomb/integrations/rack_spec.rb

```diff
@@ -2,14 +2,26 @@
 
 if defined?(Honeycomb::Rack)
   require "rack/test"
-  require "rack/lobster"
+  begin
+    require "rack/lobster"
+  rescue LoadError
+    require "rack/session"
+    require "rackup"
```

Comment:
on second thought, I'm not sure if this line is needed. @robbkidd ?

Reply:
🤔 I think you may be right. I will check!

Reply:
It wasn't ✨ Just pushed a commit removing the duplicate "rackup" and also adding the rack-session dep to the appraisals file (not just directly in the .gemfile for rack_3)